frontend/src/EditUserProfile.tsx

```diff
@@ -0,0 +1,338 @@
+            <span>{userQueryResult.data.name[0]}</span>
+          </div>
+          <h3 className="scroll-m-20 text-xl tracking-tight lg:text-2xl text-slate-50 font-bold lg:mt-4 mt-2">
+            {userQueryResult.data.name}
+          </h3>
+          <h5 className="scroll-m-20 text-l tracking-tight lg:text-xl text-slate-50">
+            {userQueryResult.data.email}
+          </h5>
+          <div className="flex">
+            <h5 className="scroll-m-20 text-l tracking-tight lg:text-xl text-slate-50 mt-4 mb-4 font-bold">
+              Batch:
+            </h5>
+            <h5 className="scroll-m-20 text-l tracking-tight lg:text-xl text-slate-50 mt-4 mb-2 mx-2">
+              {batch}
+            </h5>
+          </div>
+          <div className="flex lg:flex-row flex-col">
+            <Select onValueChange={setFirstDegree} value={firstDegree}>
+              <SelectTrigger className="w-84 bg-slate-800 border-slate-700 focus:ring-slate-800 focus:ring-offset-slate-800 text-slate-50 mt-2">
+                <SelectValue placeholder="Select a degree" />
+              </SelectTrigger>
+              <SelectContent className="bg-slate-700 border-slate-600 text-slate-50">
+                <SelectGroup>
+                  <SelectLabel>Single Degrees</SelectLabel>
+                  <SelectItem value="A1">A1: B.E. Chemical</SelectItem>
+                  <SelectItem value="A2">A2: B.E. Civil</SelectItem>
+                  <SelectItem value="A3">
+                    A3: B.E. Electrical & Electronics
+                  </SelectItem>
+                  <SelectItem value="A4">A4: B.E. Mechanical</SelectItem>
+                  <SelectItem value="A5">A5: B. Pharmacy</SelectItem>
+                  <SelectItem value="A7">A7: B.E. Computer Science</SelectItem>
+                  <SelectItem value="A8">
+                    A8: B.E. Electronics & Instrumentation
+                  </SelectItem>
+                  <SelectItem value="AA">
+                    AA: B.E. Electronics & Communication
+                  </SelectItem>
+                </SelectGroup>
+                <SelectGroup>
+                  <SelectLabel>Dual Degrees</SelectLabel>
+                  <SelectItem value="B1">B1: M.Sc. Biology</SelectItem>
+                  <SelectItem value="B2">B2: M.Sc. Chemistry</SelectItem>
+                  <SelectItem value="B3">B3: M.Sc. Economics</SelectItem>
+                  <SelectItem value="B4">B4: M.Sc. Mathematics</SelectItem>
+                  <SelectItem value="B5">B5: M.Sc. Physics</SelectItem>
+                </SelectGroup>
+              </SelectContent>
+            </Select>
+            {firstDegree.includes("B") && secondDegree === null && (
+              <Select onValueChange={setSecondDegree}>
+                <SelectTrigger className="w-84 lg:mx-4 bg-slate-800 border-slate-700 focus:ring-slate-800 focus:ring-offset-slate-800 text-slate-50 mt-2">
```

Comment:
Based on my previous suggestion about screen sizes, you will have to edit the x-margin to show up on small and medium screens as well.
```suggestion
                <SelectTrigger className="w-84 sm:mx-4 bg-slate-800 border-slate-700 focus:ring-slate-800 focus:ring-offset-slate-800 text-slate-50 mt-2">
```

Reply:
Done in [18609f1](https://github.com/crux-bphc/chronofactorem-rewrite/pull/117/commits/18609f1f5d8a202c1be08e707cf7a730855610e1)